packages/atomic/src/components/search/atomic-sort-expression/atomic-sort-expression.ts

```diff
@@ -62,29 +67,57 @@ export class AtomicSortExpression {
    * ```
    * If you don't set this property, the sort expression can be displayed on any tab. Otherwise, the sort expression won't be displayed on any of the specified tabs.
    */
-  @ArrayProp()
-  @Prop({reflect: true, mutable: true})
-  public tabsExcluded: string[] | string = '[]';
+  @property({
+    type: Array,
+    reflect: true,
+    converter: arrayConverter,
+    attribute: 'tabs-excluded',
+  })
+  public tabsExcluded: string[] = [];
 
-  public render() {
+  @state() public error!: Error;
+
+  constructor() {
+    super();
+
+    new ValidatePropsController(
+      this,
+      () => ({
+        label: this.label,
+        expression: this.expression,
+      }),
+      new Schema({
+        label: new StringValue({required: true, emptyAllowed: false}),
+        expression: new StringValue({required: true, emptyAllowed: false}),
+      })
+    );
+  }
+
+  render() {
     if (this.tabsIncluded.length > 0 && this.tabsExcluded.length > 0) {
       console.warn(
-        'Values for both "tabs-included" and "tabs-excluded" have been provided. This is could lead to unexpected behaviors.'
+        'Values for both "tabs-included" and "tabs-excluded" have been provided. This could lead to unexpected behaviors.'
       );
     }
 
     const dropdownComponent = 'atomic-sort-dropdown';
 
-    if (!this.host.closest(dropdownComponent)) {
+    if (!this.closest(dropdownComponent)) {
       const error = new Error(
-        `The "${this.host.nodeName.toLowerCase()}" component has to be used inside an ${dropdownComponent} element.`
-      );
-      return (
-        <atomic-component-error
-          element={this.host}
-          error={error}
-        ></atomic-component-error>
+        `The "${this.nodeName.toLowerCase()}" component has to be used inside an ${dropdownComponent} element.`
       );
+      return html`<atomic-component-error
+        .element=${this}
+        .error=${error}
+      ></atomic-component-error>`;
     }
+
+    return html`<slot></slot>`;
```

Comment:
Not a big deal, but I don't see a reason for returning a slot here: the component is the Light DOM.

We could just return `nothing` instead.

If you apply this don't forget to update the tests accordingly